### DISCUSSION TITLE
FSDP and Gradient Clipping

### DISCUSSION THREAD

**Discussion by PedroRASB** [Lightning-AI/pytorch-lightning#15047] [DDP / multi-GPU / multi-node]
Hello, 
I am trying to use gradient clipping (setting gradient_clip_val in the lightning Trainer) with fsdp, but it returns an error: "pytorch_lightning.utilities.exceptions.MisconfigurationException: `gradient_clip_algorithm='norm'` is currently not supported for `FullyShardedNativeMixedPrecisionPlugin`".

Are there plans to implement gradient clipping with fsdp in Pytorch Lightning?

Thank you.
